### DISCUSSION TITLE
Anyone have an ESP32-S3-BOX?

### DISCUSSION THREAD

**Discussion by dac2020** [arendst/Tasmota#21004] [General]
I've had one of these for several weeks and unable to use it. 
I cannot get anywhere with this and sad to see it sitting on my desk doing nothing. 

Wondering if there are forums where people are willing to take on a request for payment. 
Would $100 be a reasonable payment, to you or this project perhaps as I'd like to see Tasmota continue on.

**Comment by dac2020** [comment]
Alright, so trying to offer what I think is called a bounty isn't getting me anywhere. Am I posting this in the wrong place maybe? Does anyone know where I might be able to get help with this? Or, to know when the template page or HASPmota page have been updated so I can try again on my own?

Trying HASPmota leads to errors;

```
00:00:00.098 TAP: Loaded Tasmota App 'haspmota_demo.tapp'
00:00:00.852 LVG: Could not start Universal Display
00:00:00.854 BRY: failed to run compiled code 'value_error' - native constructor cannot return NULL
```

Not sure what to do and no one seems to own one of these.

**Comment by dacCR** [comment]
Just checking if anyone else has bought and using this device yet?

**Reply by gemu2015** [reply]
don't know which version you have. i have an s3 box in use since it came up (version 1) here is the display.ini and the gpio setup. 
i use it with sound io and an externally mounted SD card


[Archiv.zip](https://github.com/arendst/Tasmota/files/14814280/Archiv.zip)

**Comment by dacCR** [comment]
Hi, thank you for this help. 
I don't see any information on the device itself, or in the console other than; HDW: ESP32-S3 v0.2 (PSRAM)

I assume it would show something like hardware version.

I do see this in the Information page but it seems to be just the ESP chip. 
ESP Chip Id	5947248 (ESP32-S3 v0.2)

How can you tell what version you have?

**Reply by gemu2015** [reply]
this is my device, no longer available
I do not know what they have changed to new version (v3)

https://github.com/espressif/esp-box/blob/master/docs/hardware_overview/esp32_s3_box_lite/hardware_overview_for_lite.md

**Reply by dacCR** [reply]
Well, thank you anyhow, it's very appreciated.
